tests/src/JIT/Performance/CodeQuality/Span/SpanBench.cs

```diff
@@ -2,467 +2,1586 @@
                 {
-                    TestFillAllArray(a);
+                    for (int i = 0; i < FillAllIterations; i++)
+                    {
+                        TestFillAllArray(a);
+                    }
                 }
             }
         }
-    }
 
-    [Benchmark]
-    public static void FillAllReverseSpan()
-    {
-        byte[] a = new byte[Size];
-        Span<byte> s = new Span<byte>(a);
-        foreach (var iteration in Benchmark.Iterations)
+        [Benchmark]
+        public static void FillAllReverseSpan()
         {
-            using (iteration.StartMeasurement())
+            byte[] a = new byte[Size];
+            Span<byte> s = new Span<byte>(a);
+            foreach (var iteration in Benchmark.Iterations)
             {
-                for (int i = 0; i < Iterations; i++)
+                using (iteration.StartMeasurement())
                 {
-                    TestFillAllReverseSpan(s);
+                    for (int i = 0; i < FillAllIterations; i++)
+                    {
+                        TestFillAllReverseSpan(s);
+                    }
                 }
             }
         }
-    }
 
-    [Benchmark]
-    public static void FillAllReverseArray()
-    {
-        byte[] a = new byte[Size];
-        foreach (var iteration in Benchmark.Iterations)
+        [Benchmark]
+        public static void FillAllReverseArray()
         {
-            using (iteration.StartMeasurement())
+            byte[] a = new byte[Size];
+            foreach (var iteration in Benchmark.Iterations)
             {
-                for (int i = 0; i < Iterations; i++)
+                using (iteration.StartMeasurement())
                 {
-                    TestFillAllReverseArray(a);
+                    for (int i = 0; i < FillAllIterations; i++)
+                    {
+                        TestFillAllReverseArray(a);
+                    }
                 }
             }
         }
-    }
-
-    [Benchmark]
-    public static void QuickSortSpan()
-    {
-        int[] data = new int[Size];
-        int[] unsortedData = GetUnsortedData();
-        Span<int> span = new Span<int>(data);
 
-        foreach (var iteration in Benchmark.Iterations)
+        [Benchmark]
+        public static void QuickSortSpan()
         {
-            using (iteration.StartMeasurement())
+            int[] data = new int[Size];
+            int[] unsortedData = GetUnsortedData();
+            Span<int> span = new Span<int>(data);
+
+            foreach (var iteration in Benchmark.Iterations)
             {
-                for (int i = 0; i < Iterations; i++)
+                using (iteration.StartMeasurement())
                 {
-                    Array.Copy(unsortedData, data, Size);
-                    TestQuickSortSpan(span);
+                    for (int i = 0; i < QuickSortIterations; i++)
+                    {
+                        Array.Copy(unsortedData, data, Size);
```

Comment:
Why are we doing the copy inside the timing loop?  Seems like we would just want to be running the actual quicksort algorithm.